packages/block-editor/src/components/border-radius-control/index.js

```diff
@@ -104,6 +104,7 @@ export default function BorderRadiusControl( { onChange, values } ) {
 							units={ units }
 						/>
 						<RangeControl
+							__next40pxDefaultSize
```

Comment:
Can be seen in the block inspector ▸ Styles ▸ Border & Shadow of an Image block. No visual change because this is a standalone slider-only RangeControl instance.

<img src="https://github.com/user-attachments/assets/aa850e0a-4884-4f22-bf4b-0ba226d64205" alt="BorderRadiusControl" width="260">